builder/virtualbox/ovf/builder.go

```diff
@@ -48,6 +51,20 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 	state.Put("hook", hook)
 	state.Put("ui", ui)
 
+	// Convert source path to downloadable url
+	sourcePath, err := common.DownloadableURL(b.config.SourcePath)
+	if err != nil {
+		return nil, fmt.Errorf("Error preparing source_path: %s", err)
+	}
+
+	// Unless already specified, override the targetPath whenever dealing with OVAs
```

Comment:
Thanks for the feedback. Believe it or not, I knew the cache was set up correctly by default. 

The reason I defined the target path was because I was having a lot of trouble creating valid ova files for testing (I really shouldn't have been trying in the first place). At some point I became convinced the only way to create an ova file was if everything shared the same base name as the ovf. Thus, I was forcing the name to be the same. However, after taking a closer look at [the ovf format specification](https://www.dmtf.org/sites/default/files/standards/documents/DSP0243_2.1.1.pdf), I've realized all my woes were the result of my failure to follow the ordering defined on page 15.

I'll clear out all code that is unnecessary. 

Cheers!